apps/gitness/src/routes.tsx

```diff
@@ -107,7 +97,8 @@ export const repoRoutes: CustomRouteObject[] = [
             element: <RepoSummaryPage />,
             handle: {
               breadcrumb: () => <Text>Summary</Text>,
-              routeName: RouteConstants.toRepoSummary
+              routeName: RouteConstants.toRepoSummary,
+              documentTitle: ({ repoId }: { repoId: string }) => `Repository | ${repoId} | Summary`
```

Comment:
if every page needs to provide the complete path, it seems to defeat the purpose. could we have something like this:

```
{
  path: 'repos',
  handle: {
    title: 'Repository'
  },
  children: [
   {
      path: ':id',
      handle: {
        title: (id) => id
      }
    } 
  ]
}
```

and we could combine all segments upto the matching path dynamically.

Reply:
Done.